container/templates/build-docker-compose.j2.yml

```diff
@@ -18,5 +18,6 @@ ansible-container:
     - {{ base_path }}:/ansible-container/:Z
     {% if with_volumes %}{% for vol in with_volumes %}
     - {{ vol }}{% endfor %}{% endif %}
+    {% if params.playbook %}- {{ params.playbook }}:/ansible-container{{ params.playbook }}:Z {% endif %}
```

Comment:
> Also, speaking of permissions, @dustymabe needs to review the use of :Z here. It's my understanding we may not want to continue using :Z. I'm not an SELinux expert, so I may be wrong. He should at least be aware of it though.

the use of `:Z` should be fine in this case as it is consistent with what is done elsewhere right now in ansible-container. As part of https://github.com/ansible/ansible-container/issues/168 we may overhaul all of the uses of `:Z` and replace with some conditional logic to determine if `:Z` should be used or not.